### ISSUE TITLE
Describe setup process and options available when configuring a new analysis profile

### ISSUE THREAD

**Issue by mhkc** [genomic-medicine-sweden/JASEN#57]
A analysis profile is used to tweak the execution of the pipeline to tailor it to different species etc

The setup process is currently rather conveluted...

**Comment by ryanjameskennedy** [comment]
Agreed, documentation needs to be more explicit with everything you can or need to change.
